### DISCUSSION TITLE
Stats History auto-refresh?

### DISCUSSION THREAD

**Discussion by lavrentius-dev** [webmin/webmin#2343] [Q&A] [answered]
I noticed that the graphs in the Stats History panel seem to auto-refresh for a few minutes only, after that they seem to freeze and I need to either switch between the Webmin and Dashboard panels or to refresh the browser page for the graphs to be updated. Is this the expected behaviour?

**Comment by iliaross** [comment, marked as answer]
> Is this the expected behaviour?

Yes, it does freeze if the browser window is not focused. However, I might need to double-check the timing.

Could you share more about your workflow? It only freezes when you switch applications or move to another browser tab, correct?

**Reply by lavrentius-dev** [reply]
That's right, in a desktop browser so I guess it's fine as it resumes after a delay. On an Android tablet, the browser app freezes when I keep the screen on for a few minutes with the Webmin tab open, but I suspect this is a different issue related to mobile rather than Webmin.
